vendor.conf

```diff
@@ -4,7 +4,7 @@ github.com/asaskevich/govalidator f9ffefc3facfbe0caee3fea233cbb6e8208f4541
 github.com/Azure/go-ansiterm d6e3b3328b783f23731bc4d058875b0371ff8109
 github.com/beorn7/perks 3a771d992973f24aa725d07868b467d1ddfceafb
 github.com/containerd/console c12b1e7919c14469339a5d38f2f8ed9b64a9de23
-github.com/containerd/containerd e6b3f5632f50dbc4e9cb6288d911bf4f5e95b18e # v1.2.4
+github.com/containerd/containerd a15b6e2097c48b632dbdc63254bad4c62b69e709
```

Comment:
Is it a version used transitively by moby/moby? Is there any associated release?

Reply:
Had that discussion a couple of times as well; basically, the Go client of containerd does not have a stable API (see https://github.com/containerd/containerd/blob/master/RELEASES.md#public-api-stability), so using a version that's more recent than the runtime itself can be needed at times 

I still try to stick to tagged versions where possible, but in this case, some changes were needed for buildkit that are not yet in a release (perhaps a new release is done before 19.03 GA, in which case (possibly) we could pin to a tagged version)